index.ts

```diff
@@ -9,15 +9,41 @@ export interface ReplicatorMetaInput {
     }
 }
 
+const reverseAutocaptureEvent = (autocaptureEvent: any) => {
+    // TRICKY: This code basically reverses what the plugin server does
+    // Adapted from https://github.com/PostHog/posthog/blob/master/plugin-server/src/utils/db/elements-chain.ts#L105
+    const { elements, properties, ...event } = autocaptureEvent
+
+    const $elements = elements.map((el: any) => {
+        // $el_text and attributes are the only differently named parts
+        const { attributes, text,  ...commonProps } = el
+        return {
+            ...commonProps,
+            $el_text: text,
+            ...attributes
+        }
+    })
+
+    return {
+        ...event,
+        properties: {
+            ...properties,
+            $elements: $elements,
+        },
+    }
+}
+
 const plugin: Plugin<ReplicatorMetaInput> = {
     exportEvents: async (events, { config }) => {
         const batch = []
         for (const event of events) {
             // eslint-disable-next-line @typescript-eslint/no-unused-vars
-            const { team_id, now, offset, sent_at, $token, project_id, api_key, ...sendableEvent } = { ...event, token: config.project_api_key }
+            const { team_id, now, offset, sent_at, $token, project_id, api_key, ...sendableEvent } = { ...event, token: config.project_api_key } as any
```

Comment:
Typescript didn't like that some of those properties weren't in the type. I could take a deeper look but......

Reply:
Looks like a lot of these were outdated, because the plugin was on a really old version of `plugin-scaffold`. Updated so that we're up-to-date and don't need `as any`.